src/lib/items/Item.js

```diff
@@ -211,8 +245,8 @@ export default class Item extends Component {
   }
 
   mountInteract() {
-    const leftResize = this.props.useResizeHandle ? this.dragLeft : true
-    const rightResize = this.props.useResizeHandle ? this.dragRight : true
+    const leftResize = this.props.useResizeHandle ? ".rct-item-handler-resize-left" : true

```

Comment:
I am trying to get the styles out of being inline. The inline styles cause us problems because they are harder to over-ride. We end up using "!imporant" in our stylesheet.

Reply:
Sorry! What I am doing here is mounting the resize handlers to a selector of a class, rather than a DOM node. The problem I had is that if the resize handlers had children, the children did not have the resize events mounted from interactjs on them.

Reply:
maybe the problem that you are having is with the itemRenderer?
I think the resize handlers should be per dom element and not a class name. this will make it easier to implement features like multi-select in the future. I am not an expert with interactjs to be honest. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is being caused by itemRenderer and not the selectors?
I am not an expert with interactjs but I think dom element was used here to help features like multiselect. What do you think?

Reply:
maybe the problem here is with the item renderer?
to be honest I am not a big fan of css selectos but I am as well not an expert with interactjs

Reply:
maybe the problem here is with the item renderer?
to be honest I am not a big fan of css selectos but I am as well not an expert with interactjs

Reply:
This was in my renderer

```
{props.itemContext.useResizeHandle &&
  <div {...resize.left} >
     {props.itemContext.selected &&
        <Semantic.Icon
            className="rct-item-handler-resize-left"
            size="large"
            fitted
            color="black"
            name="angle double left"/>
      }
 </div>
}
```

What happens is when the item is selected, indicator arrows on the left and right appear to let users know they can resize. Because the ineractjs handler was mounted on the single element, the inner Semantic.Icon element could not be selected for resizing.

By adding the class selector to the resize, anything element under the `<div {...getItemProps()}>` div and the `rct-item-handler-resize-left` class will have the resize events attached to it.